.github/workflows/deploy.yml

```diff
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Deploy to Vercel CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: [develop]
```

Comment:
IIRC we wanted to remove these lines so we only trigger it manually. We can leave the `push` hook and see what happens :)

Reply:
Actually, we probably also want to delete the `push` hook because vercel should already watch the `develop` branch.

Reply:
> because vercel should already watch the develop branch. 


I don't think it does, because we need Pro to work with orga-repos (which is one of the reasons for this CI-workflow in the first place). Or do I miss anything?